simexpal/launch/common.py

```diff
@@ -218,12 +218,19 @@ def compile_manifest(run):
 
 def invoke_run(manifest):
 	# Create the output file. This signals that the run has been started.
-	stdout = None
-	with open(manifest.output_file_path('out'), "w") as f:
-		# We do not actually need to write anything to the output file.
-		# However, we might want to pipe experimental output to it.
-		if manifest.output == 'stdout':
+	(stdout_pipe, stdout) = (None, None)
+	if manifest.output == 'stdout':
+		with open(manifest.output_file_path('out'), "w") as f:
+			# We do not actually need to write anything to the output file.
+			# However, we might want to pipe experimental output to it.
 			stdout = os.dup(f.fileno())
```

Comment:
We should still create the output file even if `output != 'stdout'`. In that case, however, we don't need to `dup` the file handle.

Reply:
Done. The check `manifest.output == 'stdout'` is done after creating the output file.